### PR TITLE
feat: Add Feeds Module to node package

### LINKED ISSUE #1324: `@novu/node`: complete the feed module
- [ ] Create
- [ ] Get
- [ ] Delete
- [ ] Update (no endpoint or not documented)
- [ ] Completed docs

Feel free to tell me if there's more I'm missing and I'll update the list.

### PR DESCRIPTION
**Pull request by viveknshah**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Completes the [feeds module](https://docs.novu.co/api/create-feed/ for @novu/node.
- **Why was this change needed?** (You can also link to an open issue here)
This resolves #1324 
- **Other information**:

### CODE REVIEW

File: packages/node/src/lib/feeds/feeds.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { WithHttp } from '../novu.interface';
```
Comment (florian-lefebvre) [inline packages/node/src/lib/feeds/feeds.ts:1]:
```suggestion
import axios from 'axios';
import { WithHttp } from '../novu.interface';
```
Even if not use, I believe it's required in order to work with `WithHttp`

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]
Great work! I've only noticed a small change

**Review by florian-lefebvre — APPROVED** [review]

**Comment by viveknshah** [comment]
Have updated the docs in https://github.com/novuhq/api-docs/pull/20

File: packages/node/src/lib/feeds/feeds.spec.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { Novu } from '../novu';
+import axios from 'axios';
+
+const mockConfig = {
+  apiKey: '1234',
+};
+
+jest.mock('axios');
+
+describe('test use of novus node package - Feeds class', () => {
```
Comment (p-fernandez) [inline packages/node/src/lib/feeds/feeds.spec.ts:10]:
```suggestion
describe('Use of Novu node package - Feeds class', () => {
```

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**